src/azul/terra.py

```diff
@@ -150,9 +150,9 @@ def register_with_sam(self) -> None:
             log.info('Google service account previously registered with SAM.')
         elif (response.status == 500
               and 'Cannot update googleSubjectId' in response.data.decode()):
-            log.warning('Unable to register %s. SAM does not allow re-registration of '
-                        'service account emails. Refer to the troubleshooting section '
-                        'of the README.', self.credentials.service_account_email)
+            raise RequirementError(f'Unable to register {self.credentials.service_account_email}. '
```

Comment:
trimmed line length cannot exceed 81 chars, this line is at 87. 
https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#line-length